code/_onclick/telekinesis.dm

```diff
@@ -32,18 +32,24 @@ var/const/tk_maxrange = 15
 		return
 
 	var/obj/item/tk_grab/O = new(src)
-	user.put_in_active_hand(O)
-	O.host = user
-	O.focus_object(src)
+	if(user.put_in_hands(O))
+		user.put_in_active_hand(O)
```

Comment:
I think it was put_in_active_hand only. I was cycling through all the available put in hand procs until I found one that returned false, so maybe I don't even need the put_in_active_hand. I'll check.